swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -0,0 +1,157 @@
+// Ready translated locale messages
+
+const translations = {
+  en: {
+    message: {
+      index: {
+        logIn: "Log In",
+      },
+      error: {
+        frontPage: "To the Front Page",
+        Unauthorized: "401 – Not logged in",
+        Unauthorized_text: "The action you want to take requires logging" +
+                           "in. You can do so with the button below.",
+        Notfound: "404 – Could not find the page you were looking for.",
+        Notfound_text: "The front page, however, can be found – in the link " +
+                       "below.",
+        Forbidden: "403 – Wait, thats forbidden!",
+        Forbidden_text: "You are forbidden from doing the previous request. " +
+                        "If you are sure you should be allowed to perform " +
+                        "said operation, contact the service administrator. " +
+                        "Otherwise head back to the front page from the " +
+                        "button below.",
+      },
+      program_name: "Object Browser",
+      currentProj: "Current project",
+      logOut: "Log Out",
+      cscOrg: "CSC - IT Center For Science LTD",
+      devel: "developed by",
+      table: {
+        name: "Name",
+        objects: "Objects",
+        size: "Size",
+        modified: "Last Modified",
+        paginated: "Paginated",
+        pageNb: "per page",
+        fileHash: "Hash",
+        fileType: "Type",
+        fileDown: "File Download",
+      },
+      dashboard: {
+        prj_usage: "Project usage",
+        account: "Account",
+        containers: "Containers",
+        objects: "Objects",
+        usage: "Usage",
+        cur_billing: "Currently consumes",
+        prj_str_usag: "Project storage usage",
+        equals: "Equals",
+        more_info: "More information",
+        billing_info: "Pouta billing information",
+        quota_info: "Pouta default quotas",
+        avail_info: "Information on project billing unit availability etc.",
+        dashboard: "User information",
+        browser: "Browser",
+        tooltip_disable: "Hide tooltip",
+        hour: "hour",
+        default_notify: "The information on consumed billing units and the " +
+                        "available quota is derived from the default Pouta " +
+                        "values. If you have a separate pricing contract " +
```

Comment:
Not sure how to rephrase this but try to rephrase the sentence without `you` in it.